handbook/direction/maturity/index.md

```diff
@@ -0,0 +1,63 @@
+# Feature Maturity Tracker
+
+Sourcegraph keeps track of how mature each of our feature areas are, to help you make the best decisions about how to use our product. Clicking on the feature name will take you to its documentation.
+
+If you do not see content below, check you have JavaScript enabled then contact Product Management.
+
+## Maturity levels
+
+- ○: not yet available, but on our roadmap
+- ◔: available in the product, but may not be ready for production use yet
+- ◑: usable for primary jobs to be done, but may lack depth
```

Comment:
Is "may lack depth" gonna be true for every initiative? I can imagine that this progression may be specific to the product or team, where sometimes it makes sense to go depth-first, and sometimes breadth-first, and sometimes a different dimension entirely. For example, API docs seems to be going depth-first by focusing on one specific language first and making the experience _deep_ for that language before going _wide_ and supporting many languages. I think we should keep that flexibility for different initiatives.

Reply:
Good point, added 'or breadth'.

```suggestion
- ◑: usable for primary jobs to be done, but may lack depth and/or breadth
```